src/fuzz_introspector/datatypes/function_profile.py

```diff
@@ -50,6 +50,12 @@ def __init__(self, elem: Dict[Any, Any]) -> None:
         self.constants_touched = elem['constantsTouched']
         self.branch_profiles = self.load_func_branch_profiles(elem['BranchProfiles'])
 
+        # Duplication of functions_reached to keep the original set of call tree
+        # relationship for further processing and analysis. This could avoid lost
+        # of callpath tree information when functions_Reached is further propogated
+        # by later operations.
```

Comment:
Could you go over this text again? There are a few typos (propogated --> propagated, "functions_Reached" --> functions_reached

Reply:
Done